homebrew_challenge/src/components/SingleValue/SingleValueBar.css

```diff
@@ -97,3 +103,9 @@
     font-size: 2rem;
   }
 }
+
+@media screen and (max-width: 290px) {
```

Comment:
Isn't this duplicated ?

Reply:
It's an unfinished business between me and the website. Will be sorted today!

Reply:
This still looks the same as the block at line 78

Reply:
Just finished it, all done!